app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+## Psychology 
+
+As a psychology teacher you'll help pupils develop an understanding of the ideas and values that characterise ‘self’ and others. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=C8&visa_status=false">Search for psychology teacher training courses</a>.
+
+## Religious education 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=religious_group&button=#funding-widget">Tax-free bursaries of £10,000 are available for eligible trainee religious education teachers</a>. 
+
+As a religious education teacher you'll play a part in your pupils spiritual and moral development and prepare them for experiences of later life. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/religious-education-secondary-ks3-l/units">explore what teaching a religious education lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=V6&visa_status=false">Search for religious education teacher training courses</a>.
+
+## Russian 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=languages&button=#funding-widget">Tax-free bursaries of £25,000 are available for all eligible trainee language teachers</a>.
+
+As a Russian teacher you'll help pupils learn a new language and open them up to a different culture.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&send_courses=false&study_type%5B%5D=full_time&study_type%5B%5D=part_time&subjects%5B%5D=21&visa_status=false">Search for Russian teacher training courses</a>.
+
+## Science 
+
+As a science teacher you'll perform experiments with pupils and teach them physics, chemistry and biology.
+
+You can <a href="https://www.thenational.academy/teachers/programmes/science-secondary-ks3-l/units">explore what teaching a science lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=F0&visa_status=false">Search for science teacher training courses</a>.
+ 
+## Social sciences
+
+As a social sciences teacher you'll be able to teach sociology, politics, psychology and philosophy to pupils at key stage 4.  
```

Comment:
@caitlin-reidy what's the USP for this course? how would it be preferable to say just teaching one of these subjects? what would inspire someone to want to teach this?